source/objects/lander.lua

```diff
@@ -71,25 +130,45 @@ end
 
 
 local function thrustLeft(lander, dt)
-	if Lander.hasUpgrade(lander, Modules.sideThrusters) then
-		local forceX = 0.5 * dt		--!
+-- TODO: consider the side thrusters moving left/right based on angle and not just movement on the X axis.
+	if Lander.hasUpgrade(lander, Modules.sideThrusters) and landerHasFuelToThrust(lander, dt) then
+		local forceX = 0.5 * dt
 		lander.vx = lander.vx - forceX
 		-- opposite engine is on
 		lander.rightEngineOn = true
 		lander.fuel = lander.fuel - forceX
 	end
+	
+	-- if trying to side thrust and has parachute and descending and on the screen then ...
+	if Lander.hasUpgrade(lander, Modules.parachute) and not landerHasFuelToThrust(lander, dt) then
+		if lander.vy > 0 and lander.y > 15 then		-- 15 is enough to clear the fuel gauge
+			-- parachutes allow left/right drifting even if no fuel and thrusters available
+			deployParachute(lander)
```

Comment:
Wouldn't it be better to deploy the parachute automatically or when pressing up/w? What if you bought the parachute but not the side thrusters, that would mean the player couldn't use the parachute at all I believe.

Reply:
It re uses the side keys so you can steer the lander left and right. You don't need thrusters to steer the parachute. I'll see what play testers think about auto-deployment.